include/NovelRT.h

```diff
@@ -0,0 +1,112 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_H
+#define NOVELRT_H
+
+#include <math.h>
```

Comment:
nit: might be nice to sort these. Maybe:
* stdlib includes
* glad includes
* glm includes
* sdl2 includes
* spdlog includes

would also be good to use `cmath` rather than `math.h`